include/revng/Model/TypeDefinitionKind.h

```diff
@@ -5,4 +5,29 @@
 //
 
 #include "revng/Model/Generated/Early/TypeDefinitionKind.h"
+
+namespace model::TypeDefinitionKind {
+
+constexpr llvm::StringRef automaticNamePrefix(Values V) {
```

Comment:
This PR doesn't change any default generated name in C/disambiguation prefix for `CustomName` that conflict, right?

Reply:
No, I didn't touch the names, only the places where they are encoded.